pkgdb/src/buildenv/realise.cc

```diff
@@ -731,6 +744,42 @@ makeActivationScripts( nix::EvalState &         state,
   return { realised, references };
 }
 
+std::pair<buildenv::RealisedPackage, nix::StorePathSet>
+makePackageBuildScripts( nix::EvalState &         state,
+                         const BuildenvLockfile & lockfile )
+{
+  auto tempDir          = std::filesystem::path( nix::createTempDir() );
+  auto packageBuildsDir = tempDir / PACKAGE_BUILDS_SUBDIR_NAME;
+  std::filesystem::create_directories( packageBuildsDir );
+
+
+  if ( lockfile.manifest.build.has_value() )
```

Comment:
question nonblocking: should we completely skip adding if the manifest doesn't define any builds?

Reply:
I think this is a bit closer to the behavior of `makeActivationScripts`